kDriveCore/Data/UploadQueue/Servicies/FreeSpace/FreeSpaceService.swift

```diff
@@ -69,41 +73,107 @@ public struct FreeSpaceService {
     }
 
     /// Run cache consistency checks
+    ///
+    /// Removes orphan import files, and cache if constrained in space
+    /// This works on the assumption that the UploadQueue is stopped
     public func auditCache() {
-        cleanOrphanFiles()
+        assert(uploadQueue.operationQueue.isSuspended, "expecting the uploadQueue to be suspended")
+
+        cleanOrphanImportFolderFiles()
+        cleanOrphanRootImportFiles()
         cleanCacheIfAlmostFull()
     }
 
+    /// Check for legacy imports files, clean if file is not tracked in DB.
+    ///
+    /// This checks within the appGroup directory, not in the import one.
+    private func cleanOrphanRootImportFiles() {
```

Comment:
This removes folders containing files to import, no longer importing, present at the root of the `appGroup` directory.